wooey/django_compat.py

```diff
@@ -16,21 +22,23 @@ def __init__(self, content, mimetype='application/json', status=None, content_ty
                 status=status,
                 content_type=content_type,
             )
+
+
 else:
     from django.http import JsonResponse
 
 if DJANGO_VERSION >= DJ18:
+
     from django.template import Engine
+    from django.contrib.contenttypes.fields import GenericForeignKey
```

Comment:
Can you move this to a separate place? I'd like to keep each segment specific to a given task -- like above this is 'Compatibilty for JSON', this was just 'compatibility for Template Rendering', etc.